src_MFsurfchem/MFsurfchem_functions.H

```diff
@@ -18,10 +18,10 @@ using namespace common;
 
 void InitializeMFSurfchemNamespace();
 
-void init_surfcov(MultiFab& surfcov, const amrex::Real* dx);
+void init_surfcov(MultiFab& surfcov, const amrex::Geometry geom);
```

Comment:
Change all of these instances of "Geomtry" to "Geometry&" in each of the 3 functions declarations (both here and in the .cpp file).  This is very minor but it prevents AMReX from making a copy of the geometry object, and instead accesses the passed-in geometry object from its pointer.

Reply:
Sounds good. Will do!